composer.json

```diff
@@ -67,8 +69,13 @@
         "lint": "parallel-lint src tests",
         "phpunit": "phpunit --verbose --colors=always",
         "phpcs": "phpcs src tests --standard=psr2 -sp --colors",
+        "phpstan": [
+            "@php vendor/phpstan/phpstan-shim/phpstan.phar analyse -c phpstan.neon src --level 2 --no-progress",
```

Comment:
by splitting analysis of `src` and `tests` separate config file can be provided (e.g. with different ignored erros)